### PR TITLE
feat(shield): list users order by

### PR DESCRIPTION
**Pull request by mabdh**
- adding sort and order by clause in list users in shield

### CODE REVIEW

File: gotocompany/shield/v1beta1/shield.proto [hunk 1]
```diff
@@ -584,6 +584,8 @@ message ListUsersRequest {
   int32 page_size = 1;
   int32 page_num = 2;
   string keyword = 3;
+  string sort = 4;
+  string order = 5;
```
Comment (rahmatrhd) [inline gotocompany/shield/v1beta1/shield.proto:588]: it's a bit hard to determine which one is for column name which one for order direction

`order_by` (for column name_ and `sort` (asc | desc) would be better I guess

Comment (mabdh) [reply gotocompany/shield/v1beta1/shield.proto:588]: no I think the `sort` could be kept for column name as sort by.
just the direction could be configured to be clearer. probably we could call it `direction` like how compass does.
wdyt?

Comment (rahmatrhd) [reply gotocompany/shield/v1beta1/shield.proto:588]: `direction` or `sort_direction` sounds good to me

Comment (mabdh) [reply gotocompany/shield/v1beta1/shield.proto]: updated to `direction`

**Pull request merged**